KomaMRICore/src/rawdata/ISMRMRD.jl

```diff
@@ -85,7 +78,7 @@ julia> plot_signal(raw)
 """
 function signal_to_raw_data(
     signal, seq;
-    phantom_name="Phantom", sys=Scanner(), sim_params=Dict{String,Any}()
+    phantom_name="Phantom", sys=Scanner(), sim_params=Dict{String,Any}(), ndims=2
```

Comment:
Remember to merge with original branch to not steal the credit.

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/176d3071b3337e6e17ecc2280adb4e6512c73367. Your commits are now incorporated.

Reply:
Doing it like that my changes do not appear as my changes in the git blame, do it the other way around (properly).

Reply:
In the end, it was necessary for the owner, @cncastillo, to manually update the changes in the avoid-circular-deps branch. Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/240/commits/d2fd052d1c24398d3d74090f1bb89103a4c1ebc6.